src/main/java/roomescape/service/AuthorizationService.java

```diff
@@ -0,0 +1,42 @@
+package roomescape.service;
+
+import org.springframework.stereotype.Component;
+import roomescape.dao.MemberDao;
+import roomescape.dto.request.LoginRequest;
+import roomescape.dto.response.MemberNameResponse;
+import roomescape.entity.AccessToken;
+import roomescape.entity.Member;
+import roomescape.exception.InvalidAccessTokenException;
+import roomescape.exception.MemberNotFoundException;
+import roomescape.web.LoginMember;
+
+@Component
+public class AuthorizationService {
```

Comment:
서비스의 경우 관례적으로 `@Service` 어노테이션을 사용합니다.

```suggestion
@Service
public class AuthorizationService {
```

Reply:
맞습니다! 처음엔 @Component와 @Service가 실질적으로 다르지 않아 바꾸지 않았는데, 개발자 간 관례를 지키는 것도 중요한 것 같아요.

수정했습니다!